src/StdChains.sol

```diff
@@ -199,12 +199,20 @@ abstract contract StdChains {
         setChainWithDefaultRpcUrl(
             "mainnet", ChainData("Mainnet", 1, "https://eth-mainnet.alchemyapi.io/v2/pwc5rmJhrdoaSEfimoKEmsvOjKSmPDrP")
         );
+        // @notice Goerli is to be sunset by April 3, 2024
+        // @depreciated goerli
         setChainWithDefaultRpcUrl(
             "goerli", ChainData("Goerli", 5, "https://goerli.infura.io/v3/b9794ad1ddf84dfb8c34d6bb5dca2001")
         );
         setChainWithDefaultRpcUrl(
             "sepolia", ChainData("Sepolia", 11155111, "https://sepolia.infura.io/v3/b9794ad1ddf84dfb8c34d6bb5dca2001")
         );
+        setChainWithDefaultRpcUrl(
+            "holesky", ChainData("Holesky", 17000, "https://rpc.holesky.ethpandaops.io")
+        );
+
+
+
```

Comment:
needs `forge fmt` please!

Reply:
Amended previous commit and fixed formatting